x-pack/docs/en/security/authentication/configuring-ldap-realm.asciidoc

```diff
@@ -206,6 +206,7 @@ xpack:
       realms:
         ldap:
           ldap1:
+            order: 0
```

Comment:
There's another line in this file that I think should be changed: 

> If you are configuring multiple realms, you should also explicitly set the `order` attribute ....

I think it should be changed to "you must also explicitly set the `order` attribute".

Reply:
Good catch. Will update. Thanks